### PR TITLE
[LIMINAL-25] Respond with 204 to favicon requests to python server

### PR DESCRIPTION
**Pull request by simlev**
Added route /favicon.ico that returns http status code 204 No content

https://issues.apache.org/jira/browse/LIMINAL-25

### CODE REVIEW

**Comment by aviemzur** [comment]
Some clients request file `favicon.ico` when calling URLs, this request fails

**Comment by aviemzur** [comment]
LGTM
Please add unit test which verifies we get 204 for favicon request

**Comment by aviemzur** [comment]
@simlev any luck with unit test?

**Comment by simlev** [comment]
@aviemzur I added test. 

There is a strange conflict now in the branch.

File: liminal/build/image/python_server/liminal_python_server.py [hunk 1]
```diff
@@ -45,6 +45,12 @@ def __get_endpoint_function(endpoint_config):
 
     blueprint = Blueprint('liminal_python_server_blueprint', __name__)
 
+
+    @blueprint.route('/favicon.ico', methods=('GET', 'POST'))
+    def no_content():
+        return '', 204
+
+
```
Comment (aviemzur) [inline liminal/build/image/python_server/liminal_python_server.py:53]:
```suggestion
```

File: liminal/build/image/python_server/liminal_python_server.py [hunk 2]
```diff
@@ -53,6 +59,7 @@ def show(endpoint):
         else:
             return 'Page not found.', 404
 
+
```
Comment (aviemzur) [inline liminal/build/image/python_server/liminal_python_server.py:62]:
```suggestion
```

File: tests/runners/airflow/build/http/python/test_python_server_image_builder.py [hunk 3]
```diff
@@ -96,6 +96,11 @@ def __test_build_python_server(self, use_pip_conf=False, python_version=None):
 
         self.assertEqual(f'Input was: {json.loads(json_string)}', server_response)
 
+        server_favicon_response = urllib.request.urlopen(
```
Comment (aviemzur) [inline tests/runners/airflow/build/http/python/test_python_server_image_builder.py:99]:
```suggestion
        server_favicon_response = urllib.request.urlopen('http://localhost:9294/favicon.ico')
```

File: tests/runners/airflow/build/http/python/test_python_server_image_builder.py [hunk 4]
```diff
@@ -96,6 +96,11 @@ def __test_build_python_server(self, use_pip_conf=False, python_version=None):
...
+            'http://localhost:9294/favicon.ico')
```
Comment (aviemzur) [inline tests/runners/airflow/build/http/python/test_python_server_image_builder.py:100]:
```suggestion
```

File: liminal/build/image/python_server/liminal_python_server.py [hunk 5]
```diff
@@ -17,7 +17,7 @@
 # under the License.
 
 import yaml
-from flask import Blueprint, Flask, request
+from flask import Flask, request, Blueprint
```
Comment (aviemzur) [inline liminal/build/image/python_server/liminal_python_server.py:20]:
```suggestion
from flask import Blueprint, Flask, request
```

**Pull request merged**